### DISCUSSION TITLE
Issue Defining Multiple Vehicle Routes

### DISCUSSION THREAD

**Discussion by yibork** [eclipse/mosaic#447] [General]
Hello,

I’m encountering an issue while defining multiple routes for my vehicles in SUMO when using Mosaic. I’ve defined the routes in `name.rou.xml` and assigned them to vehicles. While the first route works as expected, whenever I add a second route, I get an error indicating that the first edge of the second route does not exist.

What’s perplexing is that even if I reuse the exact same edges from the successful first route, the error persists. 
Below are some reference images illustrating the problem:

**Image 1: Defining Routes**
 ![image](https://github.com/user-attachments/assets/5effc40e-c1e7-4b90-8cc0-d2cdf332ac27)

 **Image 2: Attempting the Same Edges for the Second Route**
![image](https://github.com/user-attachments/assets/6301c87a-3677-4d9b-8a80-c849d22a2af7)

**Image 3: The Resulting Error Message**
![image](https://github.com/user-attachments/assets/a9435f9c-c153-4102-82b9-e400f4c14224)

Is there a particular configuration or step I need to consider for defining multiple routes in Mosaic so that each vehicle can follow a different path without triggering this “first edge does not exist” error?

Any insights or suggestions would be greatly appreciated.

**Comment by schwepmo** [comment]
It seems like you resolved this issue already? If not the last image is a duplicate of the second one, a screenshot of the error message would help.
